test/suites/integration/validation_test.go

```diff
@@ -124,6 +129,39 @@ var _ = Describe("Validation", func() {
 			}
 			Expect(env.Client.Create(env.Context, nodePool)).ToNot(Succeed())
 		})
+		It("should error when minValues for a requirement key is negative", func() {
```

Comment:
Should this test also check 0 since we are also saying that you can't put 0 in that value as part of the kubebuilder spec